pkgs/development/python-modules/taiga-back/taiga-back-setup.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup, find_packages
+
+setup(
+    name = 'taiga-back',
```

Comment:
@juanfran another side-request from this nixos-pull-request: can we get a setup.py in taiga-back ?